package.json

```diff
@@ -2,6 +2,7 @@
   "name": "news-explorer-frontend",
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://www.newsexplorer.ignorelist.com",
```

Comment:
https://ian-diz.github.io/news-explorer-frontend leads to page 404. Please, deploy the app